src/lib/CRSF/CRSF.cpp

```diff
@@ -730,24 +741,24 @@ bool CRSF::UARTwdt()
             adjustMaxPacketSize();
 
             SerialOutFIFO.flush();
-#ifdef PLATFORM_ESP32
-            CRSF::Port.flush();
-            CRSF::Port.updateBaudRate(UARTrequestedBaud);
-#else
-            CRSF::Port.begin(UARTrequestedBaud);
-            #if defined(TARGET_TX_GHOST)
-            USART1->CR1 &= ~USART_CR1_UE;
-            USART1->CR3 |= USART_CR3_HDSEL;
-            USART1->CR2 |= USART_CR2_RXINV | USART_CR2_TXINV | USART_CR2_SWAP; //inverted/swapped
-            USART1->CR1 |= USART_CR1_UE;
+            #if defined(PLATFORM_ESP32) || defined(PLATFORM_ESP8266)
+                CRSF::Port.flush();
+                CRSF::Port.updateBaudRate(UARTrequestedBaud);
+            #else
+                CRSF::Port.begin(UARTrequestedBaud);
+                #if defined(TARGET_TX_GHOST)
+                USART1->CR1 &= ~USART_CR1_UE;
```

Comment:
minor! indention?

Reply:
haha yeah I was unsure about how this should be indented. With the #ifs all the way to the left it is hard to read due to being multiple levels of ifs, and it is sort of half indented now and half not? How do you suggest it should be? Let me know and I'll update.

Thanks for the other fixes, I'm sure I would have pulled my hair out for an hour before realizing there was just a typo in the ifdef.

Reply:
I am not sure either... Thatcs why the ? mark 😂
Most of the code seems to be indented so maybe you should do it here as well.
I personally prefer left aligned macros but I can live with indents as well...

Reply:
Exactly, how can this be so difficult?! I've changed it around to do the left-aligned since everything else in this file is and just modded it to be elif instead of nested ifs. How's that?

![image](https://user-images.githubusercontent.com/677183/135758081-db70078e-e05e-45e7-b18e-7b9c8a51675d.png)

Reply:
Like it! Thanks a lot 💪